types/fabric/test/index.ts

```diff
@@ -1053,3 +1053,8 @@ function sample12() {
   const x = position.left;
   const y = position.top;
 }
+
+function sample13() {
+  const rectangle = new fabric.Rect({top: 0, left: 0, width: 10, height: 10});
+  const rectangleAsHtmlCanvas: HTMLCanvasElement = rectangle.toCanvasElement();
```

Comment:
@NotWoods Thank you for your swift reply. As a test, would this be sufficient? 
I went for explicit `: HtmlCanvasElement` typing which fails on master but is passing on this branch.
Please let me know if you have any additional feedback,
thank you in advance :bow:

Reply:
Yes, this is great for a test! Thank you